chapters/03-implementation.Rmd

```diff
@@ -37,7 +37,7 @@ Given a set of sample points $X = \{x_0, x_1, ..., x_k, ..., x_{n-1}, x_n\}$ wit
 
 As described in the excellent overview by Artur Gramacki[@gramacki2018fft] simple binning or linear binning can be used, although the last is often preferred since it is more accurate and the difference in computational complexity is negligible.
```

Comment:
```suggestion
As described in the extensive overview by Artur Gramacki[@gramacki2018fft] simple binning or linear binning can be used, although the last is often preferred since it is more accurate and the difference in computational complexity is negligible.
```

Don't judge ;) "excellent" is quite subjective. Not saying that it is not possible to write _if it is truly excellent_ **objectively**. But is it?